Assets/Scripts/Authoring/PlayerMovementAuthoringComponent.cs

```diff
@@ -71,6 +71,29 @@ public class PlayerMovementAuthoringComponent : MonoBehaviour, IConvertGameObjec
         /// </summary>
         public float pushDecay = 0.0f;
 
+        /// <summary>
+        /// Duration of jumping grace period
+        ///     - Player can jump while not grounded, as long as they haven't been grounded longer than this amount of time
+        ///     - Make sure the value of jumpGraceTime < jumpCooldown. 
+        ///       Otherwise, double jumps are possible.
+        /// </summary>
+        public float jumpGraceTime = 0.0f;
+
+        /// <summary>
+        /// Time spent falling
+        /// </summary>
+        public float elapsedFallTime = 0.0f;
```

Comment:
Does this need to be a field, I'm used to elapsed fall time always starting at zero when the entity is created so this could be removed form the list of configurable fields.

Reply:
I agree that it doesn't need to be configurable, though I don't know how you want to handle / organize non-configurable fields.

Reply:
Removed from authoring component per your suggestion!